src/main/webapp/admin_dashboard_script.js

```diff
@@ -1,231 +1,264 @@
+				elementType: "geometry",
+				stylers: [{ color: "#746855" }],
+			},
+			{
+				featureType: "road.highway",
+				elementType: "geometry.stroke",
+				stylers: [{ color: "#1f2835" }],
+			},
+			{
+				featureType: "road.highway",
+				elementType: "labels.text.fill",
+				stylers: [{ color: "#f3d19c" }],
+			},
+			{
+				featureType: "transit",
+				elementType: "geometry",
+				stylers: [{ color: "#2f3948" }],
+			},
+			{
+				featureType: "transit.station",
+				elementType: "labels.text.fill",
+				stylers: [{ color: "#d59563" }],
+			},
+			{
+				featureType: "water",
+				elementType: "geometry",
+				stylers: [{ color: "#17263c" }],
+			},
+			{
+				featureType: "water",
+				elementType: "labels.text.fill",
+				stylers: [{ color: "#515c6d" }],
+			},
+			{
+				featureType: "water",
+				elementType: "labels.text.stroke",
+				stylers: [{ color: "#17263c" }],
+			},
+		],
+	});
+	//Resize Function
+	google.maps.event.addDomListener(window, "resize", function () {
+		var center = map.getCenter();
+		google.maps.event.trigger(map, "resize");
+		map.setCenter(center);
+	});
+    infoWindow = new google.maps.InfoWindow;
+	if (navigator.geolocation) {
+		navigator.geolocation.getCurrentPosition(function(position) {
+			var pos = {
+				lat: position.coords.latitude,
+				lng: position.coords.longitude
+			};
+
+			infoWindow.setPosition(pos);
+			infoWindow.setContent('This is your Neighborhood!');
+			infoWindow.open(map);
+			map.setCenter(pos);
+		}, function() {
+			handleLocationError(true, infoWindow, map.getCenter());
+		});
+	} else {
+		// Browser doesn't support Geolocation
+		handleLocationError(false, infoWindow, map.getCenter());
+	}
+}
+
+function handleLocationError(browserHasGeolocation, infoWindow, pos) {
+	infoWindow.setPosition(pos);
+	infoWindow.setContent(browserHasGeolocation ?
+		'Error: The Geolocation service failed.' :
+		'Error: Your browser doesn\'t support geolocation.');
+	infoWindow.open(map);
+}
+
+function load(file) {
+	var src = document.createElement("script");
+	src.setAttribute("type", "text/javascript");
+	src.setAttribute("src", file);
+	document.getElementsByTagName("head")[0].appendChild(src);
+}
+
+function getUserTasks() {
+	fetch("/admin-user-tasks")
+		.then((response) => response.json())
+		.then((tasks) => {
+			userTasksArray = tasks;
+			let taskSection = document.getElementById("user-tasks");
+			for (userTask of tasks) {
+				taskSection.innerHTML += addTask(userTask);
+			}
+		});
+}
+
+function getAdminTasks() {
+	fetch("/admin-tasks")
+		.then((response) => response.json())
+		.then((tasks) => {
+			adminTasksArray = tasks;
+			let taskSection = document.getElementById("admin-tasks");
+			for (adminTask of tasks) {
+				taskSection.innerHTML += addAdminTask(adminTask);
+            }
+		});
+}
+
+function addTask(task) {
+	let string = `<a href="#user-popup"><li class="admin-user-task"  onclick="openWithPopup('${task.keyString}')"><span><h3> ${task.category} </h3>`;

```

Comment:
Personally, I think that if the task changes when the mouse is hovered, it gives the feel that the whole task is a button, so it would make sense that if I click wherever on the task (except the delete button) the edit task modal would pop up.